ql/src/test/queries/clientpositive/special_character_in_tabnames_1.q

```diff
@@ -8,8 +8,8 @@ set hive.strict.checks.cartesian.product=false;
 
 -- SORT_QUERY_RESULTS
 
-create database `db~!@#$%^&*(),<>`;
-use `db~!@#$%^&*(),<>`;
+create database `db"% &()*+,-/:;<=>?[]_|{}$^!~#``@`;
```

Comment:
I would add a test case for 'db_test    ' as well. I know it will handle (even on different levels) but as we test a unit is always nice to check on Unit / domain level the characteristics of the method. Otherwise +1 from me. (It is fully optional)

Reply:
I've to admit that I don't know what 'db_test' means. I was thinking about adding a case to `standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/client/TestDatabases.java`. There is already a test that validates proper creation of the default location based on the db name, but the db name is rather simple in this case ("dummy"). I thought about adding another case that uses a db name that contains few special characters and validates that the location contains the encoded db name. Does this make sense?

Reply:
I've added an additional unit test in `TestHiveMetaStore`. It might not a perfect fit, but it appears to be a reasonable test.